frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureResultInfoPopoverCell.tsx

```diff
@@ -65,32 +60,96 @@ export const FeatureResultInfoPopoverCell = ({
-                        <LegacyPlaygroundResultFeatureStrategyList
-                            feature={feature}
-                            input={input}
-                        />
-                    </>
-                )}
+                <LegacyFeatureDetails
+                    feature={feature}
+                    input={input}
+                    onClose={() => setOpen(false)}
+                />
+                <LegacyPlaygroundResultFeatureStrategyList
+                    feature={feature}
+                    input={input}
+                />
+            </Popover>
+        </FeatureResultPopoverWrapper>
+    );
+};
+
+const DetailsPadding = styled('div')(({ theme }) => ({
+    paddingInline: `var(--popover-inline-padding, ${theme.spacing(4)})`,
+    paddingTop: theme.spacing(2.5),
+}));
+
+export const NewFeatureResultInfoPopoverCell = ({
+    feature,
+    input,
+}: FeatureResultInfoPopoverCellProps) => {
+    const [open, setOpen] = useState(false);
+    const ref = useRef(null);
+
+    const togglePopover = () => {
+        setOpen(!open);
+    };
+
+    return (
+        <FeatureResultPopoverWrapper>
+            <IconButton onClick={togglePopover}>
+                <InfoOutlined ref={ref} />
+            </IconButton>
+            <Popover
+                open={open}
+                onClose={() => setOpen(false)}
+                anchorEl={ref.current}
+                PaperProps={{
+                    sx: (theme) => ({
+                        '--popover-inline-padding': theme.spacing(4),
+                        display: 'flex',
+                        flexDirection: 'column',
+                        width: 728,
+                        maxWidth: '100%',
+                        height: 'auto',
+                        gap: theme.spacing(3),
+                        overflowY: 'auto',
+                        backgroundColor: theme.palette.background.elevation1,
+                        borderRadius: theme.shape.borderRadius,
+                    }),
+                }}
+                anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                }}
+                transformOrigin={{
+                    vertical: 'center',
+                    horizontal: 'left',
+                }}
+            >
+                <DetailsPadding>
+                    <FeatureDetails
+                        feature={feature}
+                        input={input}
+                        onClose={() => setOpen(false)}
+                    />
+                </DetailsPadding>
+                <PlaygroundResultFeatureStrategyList
+                    feature={feature}
+                    input={input}
+                />
             </Popover>
         </FeatureResultPopoverWrapper>
     );
 };
+
+export const FeatureResultInfoPopoverCell = (
+    props: FeatureResultInfoPopoverCellProps,
+) => {
+    const useNewStrategyDesign = useUiFlag('flagOverviewRedesign');
```

Comment:
Split the two versions into entirely separate components. We can consider creating a new file too, but this feels like the root of the playground changes, so I don't mind doing the split in this file alone.